### PR TITLE
WP 5.4 Compatibility: Add user Pod fields to privacy export data

### PR DESCRIPTION
**Pull request by JoryHogeveen**
Fixes: #5606

## ChangeLog

* WP 5.4 Compatibility: Add user Pod fields to privacy export data

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: classes/PodsInit.php [hunk 1]
```diff
@@ -1863,6 +1866,31 @@ public function admin_bar_links() {
 
 	}
 
+	/**
+	 * Add Pod fields to user export.
+	 * Requires WordPress 5.4+
+	 *
+	 * @since 2.7.17
+	 *
+	 * @param array   $additional_profile_data
```
Comment (tr1b0t) [inline classes/PodsInit.php]: :no_entry_sign: Missing parameter comment

<sub>hash: a4a7fe44f48db026033758cc986a61ef</sub>

File: classes/PodsInit.php [hunk 2]
```diff
@@ -1863,6 +1866,31 @@ public function admin_bar_links() {
...
+	 * @param WP_User $user
```
Comment (tr1b0t) [inline classes/PodsInit.php]: :no_entry_sign: Missing parameter comment

<sub>hash: 38a8cb64c4e380279161339bac50d502</sub>

File: classes/PodsInit.php [hunk 3]
```diff
@@ -1863,6 +1866,31 @@ public function admin_bar_links() {
...
+	 * @param array   $reserved_names
```
Comment (tr1b0t) [inline classes/PodsInit.php]: :no_entry_sign: Missing parameter comment

<sub>hash: 2a67a0690d5abbb61fb643f8828d7492</sub>

File: classes/PodsInit.php [hunk 4]
```diff
@@ -1863,6 +1866,38 @@ public function admin_bar_links() {
 
 	}
 
+	/**
```
Comment (tr1b0t) [inline classes/PodsInit.php]: :no_entry_sign: Doc comment for parameter "$additional_profile_data" missing

<sub>hash: b836cf5802aeeab8f3fe5f98ec824207</sub>

File: classes/PodsInit.php [hunk 5]
```diff
@@ -1863,6 +1866,38 @@ public function admin_bar_links() {
...
+	 * Add Pod fields to user export.
+	 * Requires WordPress 5.4+
+	 *
+	 * @since 2.7.17
+	 *
+	 * @param array    $additional_user_profile_data {
```
Comment (tr1b0t) [inline classes/PodsInit.php]: * :no_entry_sign: Expected 3 spaces after parameter type; 4 found
* :no_entry_sign: Doc comment for parameter $additional_user_profile_data does not match actual variable name $additional_profile_data

<sub>hash: e0fe3c442129825c7c827d2ef58fe3df</sub>

File: classes/PodsInit.php [hunk 6]
```diff
@@ -1863,6 +1866,38 @@ public function admin_bar_links() {
...
+	 *     An array of name-value pairs of additional user data items.  Default: the empty array.
+	 *
+	 *     @type string $name  The user-facing name of an item name-value pair, e.g. 'IP Address'.
+	 *     @type string $value The user-facing value of an item data pair, e.g. '50.60.70.0'.
+	 * }
+	 * @param WP_User  $user           The user whose data is being exported.
```
Comment (tr1b0t) [inline classes/PodsInit.php]: :no_entry_sign: Expected 1 spaces after parameter type; 2 found

<sub>hash: a9fc4977e78f937c77a57bf6f36c38f2</sub>

File: classes/PodsInit.php [hunk 7]
```diff
@@ -1863,6 +1866,38 @@ public function admin_bar_links() {
...
+	 * @param array    $reserved_names An array of reserved names.  Any item in
```
Comment (tr1b0t) [inline classes/PodsInit.php]: :no_entry_sign: Expected 3 spaces after parameter type; 4 found

<sub>hash: 8d16ccaea5f9610489b1b6e98539c145</sub>

File: classes/PodsInit.php [hunk 8]
```diff
@@ -1863,6 +1866,38 @@ public function admin_bar_links() {
...
+		$pod = pods( 'user', $user->ID );
+		if ( $pod->valid() ) {
+			foreach ( $pod->fields as $name => $field ) {
+				$additional_user_profile_data[] = array(
+					'name'  => apply_filters( 'pods_form_ui_label_text', $field['label'], $name, '', $field ),
+					'value' => $pod->display( $name ),
+				);
+			}
+		}
+		return $additional_user_profile_data;
```
Comment (sc0ttkclark) [inline classes/PodsInit.php]:
```suggestion

		if ( ! $pod->valid() ) {
			return $additional_user_profile_data;
		}

		foreach ( $pod->fields as $name => $field ) {
			$additional_user_profile_data[] = array(
				'name'  => apply_filters( 'pods_form_ui_label_text', $field['label'], $name, '', $field ),
				'value' => $pod->display( $name ),
			);
		}

		return $additional_user_profile_data;
```

**Pull request merged**